### PR TITLE
chore: document linting

### PR DESCRIPTION
**Pull request by thephez**
This PR consists only of linting/typo fixes and doesn't change any content. The goal was to improve consistency in formatting between the various documents. Linting was accomplished using the [markdownlint](https://github.com/DavidAnson/vscode-markdownlint) plugin in [Visual Studio Code](https://code.visualstudio.com/). The lint config used is also included in this PR.

## To Review
Toggle to **_Display the rich diff_** when viewing the files ([with whitespace changes hidden](https://github.com/dashpay/dips/pull/88/files?diff=unified&w=1)). Most of them will display a "No visible changes here." message which confirms that visible changes were not made. A few files have typo fixes that will be displayed.

Note: If you check the linting yourself, you'll notice some documents violate MD024 (no duplicate headers). This rule was not disabled because it's helpful to be aware of it.

### CODE REVIEW

File: .markdownlint.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "default": true,
+    "MD004": { "style": "asterisk"},
+    "MD013": false,
+    "MD026": { "punctuation": ".,;:!。，；：！" },
+    "MD033": false,
+    "MD041": false
+}
```
Comment (UdjinM6) [inline .markdownlint.json:8]: nit: Missing new line at EOF
```suggestion
}

```

**Comment by thephez** [comment]
@UdjinM6 I was thinking about wrapping the text the same in all of them too. Shouldn't make the review any harder since it doesn't change how the doc is displayed. What do you think?

**Comment by UdjinM6** [comment]
@thephez 🤷‍♂️ does this have anything to do with linting though?

**Comment by thephez** [comment]
:man_shrugging: This linter doesn't flag long lines so no, I guess.

**Review by PastaPastaPasta — APPROVED** [review]
LGTM

**Comment by thephez** [comment]
@UdjinM6 Does everything else look good here?

**Review by xdustinface — APPROVED** [review]

**Review by UdjinM6 — APPROVED** [review]
👍

**Pull request merged**